src/components/Checkbox/Checkbox.jsx

```diff
@@ -0,0 +1,12 @@
+import "./Checkbox.css";
+
+const Checkbox = (props) => {
+    return <div className="checkbox">
+        <div className={`checkbox__box ${props.checked === true ? "checkbox__box--checked" : ""}`}/>
+        <span className={`checkbox__text ${props.checked === true ? "checkbox__text--checked" : ""}`}>
```

Comment:
same as the comment above

Reply:
also you are currently not using the `checkbox__text` class. I would move the `font-size` attribute to that class as I mentioned earlier